python/lsst/obs/subaru/isr.py

```diff
@@ -247,6 +247,11 @@ def runDataRef(self, sensorRef):
                 self.saturationDetection(ccdExposure, amp)
             if self.config.doSuspect and not badAmp:
                 self.suspectDetection(ccdExposure, amp)
+            ampMask = afwImage.MaskU(ccdExposure.getMaskedImage().getMask(), amp.getRawDataBBox(),
+                                     afwImage.PARENT)
+            maskVal = ampMask.getPlaneBitMask([self.config.saturatedMaskName, self.config.suspectMaskName])
+            if numpy.all(ampMask.getArray() & maskVal > 0):
+                badAmp = True
             if self.config.doOverscan and not badAmp:
```

Comment:
I'm not sure I understand the logic here.  maskVal evaluates to 130 (2 for SAT and 128 for SUSPECT), but why are you checking that this evaluates to > 0 (and within the `numpy.all()` call)?

What is the nature of the "failure" in the overscan subtraction?  You are testing for an entirely saturated amp.  Is it true that it is guaranteed not to fail if, say, a single pixel is not saturated (but the rest are)?